accounts/views.py

```diff
@@ -803,6 +805,18 @@ def handle_uploaded_file(user_id, f):
     return True
 
 
+def copy_sound_file_to_mirror_location(source_path):
```

Comment:
there is very similar code executed after sound uploads (after describe, after csv upload and after api upload). I think this deserves a deeper refactoring so I'll leave as it is for now. I'll open an issue for that refactoring.